x-pack/plugins/infra/public/components/asset_details/hooks/use_metadata_state.ts

```diff
@@ -22,14 +22,18 @@ export function useMetadataProvider({ asset, assetType }: UseMetadataProviderPro
   const inventoryModel = findInventoryModel(assetType);
   const { sourceId } = useSourceContext();
 
-  const { loading, error, metadata } = useMetadata(
+  const { loading, error, metadata, reload } = useMetadata(
     asset.id,
     assetType,
     inventoryModel.requiredMetrics,
     sourceId,
     getDateRangeInTimestamp()
   );
 
+  const shouldRefetch = useCallback(() => {
```

Comment:
nit: `refresh` maybe? `shouldRefresh` might give the impression that this is a boolean? 🤔